nzedb/IRCClient.php

```diff
@@ -86,14 +86,14 @@
 	 * @var resource
 	 * @access protected
 	 */
-	protected $_socket = null;
+	protected $_socket;
```

Comment:
I don't see the point in removing this. Having the explicit assignment removes ambiguity and if/when we add nullable typing in the future it will make finding these easier.